node-network-aware-loading/server.js

```diff
@@ -17,7 +17,8 @@
 const express = require('express');
 const path = require('path');
 const cors = require('cors');
-const request = require('request');
+
+const IMAGES_PATH = 'public/assets/images';
```

Comment:
It looks like there's something a little strange happening with node-network-aware-loading. I've tried freshly installing all dependencies, building and starting but am failing to get the images served in dev or prod starts:

![image](https://user-images.githubusercontent.com/110953/68132458-97368d00-fed3-11e9-83eb-f89190ffc510.png)

Occasionally it seems to be missing references to dependencies, but will let you dig in deeper:

```
ReferenceError: fs is not defined
    at /Users/addyo/projects/adaptive-loading/node-network-aware-loading/server.js:68:22
```

Reply:
Sorry, `const fs = require('fs');` was missing. I fixed it.